src/main/java/com/alkemy/ong/repository/RoleRepository.java

```diff
@@ -4,7 +4,7 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
-@Repository
+@Repository //IMPORTANT - Added @Repository annotation while working on OT111-35
```

Comment:
No hace falta esta anotacion, chequear este link 
https://www.it-swarm-es.com/es/java/repository-no-es-necesario-al-implementar-jparepository/832385271/#:~:text=De%20hecho%2C%20no%20es%20necesario,de%20las%20interfaces%20predefinidas%20Repository%20.&text=Esto%20buscar%C3%A1%20Spring%20Data%20repositories,no%20est%C3%A1n%20anotados%20con%20%40Repository%20.

Reply:
Genial Osvaldo... No era mío, pero ahí lo borre.